paddle/fluid/pir/dialect/op_generator/op_infermeta_func_gen.py

```diff
@@ -613,7 +616,11 @@ def GenDistBranch(args, op_info):
         return ""
     TEMPLATE = """
   // Auto Parallel condition
-  if(!input_values.empty() && AllInputAreDist(input_values)) {{
+  if(HasDistInput(input_values)) {{
+    if(!AllInputAreDist(input_values)) {{
+        PADDLE_THROW(common::errors::Unimplemented(
+            "Current not supported mixed distributed inputs."));
```

Comment:
Mixed inputs with DenseTensor and DistDenseTensor are not supported yet.

Reply:
done, thanks!